addon/components/flexberry-file.js

```diff
@@ -1065,6 +1078,27 @@ export default FlexberryBaseComponent.extend({
     relatedModel.off('preSave', this.get('_onRelatedModelPreSave'));
   },
 
+  /**
+    Defines valid of file type.
+
+    @method _isValidTypeFile
+    @param {String} fileType file type as MIME TYPES.
+    @param {String} accept available MIME TYPES.
+    @private
+  */
+  _isValidTypeFile(fileType, accept) {
+    const isValidTypeFileCustom = this.get('isValidTypeFileCustom');
+
+    if (isValidTypeFileCustom && (typeof isValidTypeFileCustom === 'function')) {
+      return isValidTypeFileCustom();
+    }
+
+    const isFileTypeUndefined = fileType === '';
+    const isFileTypeAvailable = (!accept) || (accept && accept.indexOf(fileType) !== -1);
+
+    return (!isFileTypeUndefined && isFileTypeAvailable);
```

Comment:
Это условие не допускает неопределенный тип файла, даже если `accept` не задан.
Если `accept` не задан, должна остаться возможность выбрать файл любого типа, в том числе и неопределенного.
```suggestion
    return !isFileTypeUndefined && isFileTypeAvailable;
```